python/mxnet/test_utils.py

```diff
@@ -79,10 +79,117 @@ def random_sample(population, k):
     return population_copy[0:k]
 
 
-def rand_sparse_ndarray(shape, stype, density=None):
-    """Generate a random sparse ndarray. Returns the ndarray, value(np) and indices(np) """
+def _get_uniform_dataset_csr(num_rows, num_cols, density=0.1):
+    """Returns CSRNDArray with uniform distribution
```

Comment:
can you add a short description what we mean by uniform distribution here, similar to the one you gave for the powerlaw distribution.